library/quality/lint-baseline.xml

```diff
@@ -8,4 +8,9 @@
         <location file="src/main/AndroidManifest.xml" />
     </issue>
 
+    <issue
+        id="InvalidVectorPath"
+        message="Use 0.94 instead of .94 to avoid crashes on some devices">
+        <location file="src/main/res/drawable/fui_ic_anonymous_white_24dp.xml"></location>
+    </issue>
```

Comment:
I don't think we want this to be in our baseline since we're purposefully ignoring it. The baseline is meant to be a baseline (for lack of a better word 😆) that you're trying to slowly shrink over time.

Reply:
Do you know how to purposefully ignore a warning like this?  Can we add something in the vector XML itself?

Reply:
Yup, should be the same `tools:ignore="InvalidVectorPath"` we all know and love. 😆

Reply:
Done.